.github/workflows/e2e.yml

```diff
@@ -32,16 +32,16 @@ jobs:
       # test runtime environment, tests invoke rustc/cargo
       SYSTEM_TEST_RUST_TOOLCHAIN_VERSION: stable
 
-      # set the version of js-soroban-client to use, need to choose one of either 
+      # set the version of js-stellar-sdk to use, need to choose one of either
       # resolution options, using npm release or a gh ref:
       #
-      # option #1, set the version of soroban-js-client based on a npm release version
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION: 1.0.0-beta.2
-      # option #2, set the version of soroban-js-client used as a ref to a gh repo
-      # if a value is set on SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REPO, it takes precedence 
-      # over any SYSTEM_TEST_JS_SOROBAN_CLIENT_NPM_VERSION
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REPO:
-      SYSTEM_TEST_JS_SOROBAN_CLIENT_GH_REF:
+      # option #1, set the version of stellar-sdk based on a npm release version
+      SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION:
+      # option #2, set the version of stellar-sdk used as a ref to a gh repo if
+      # a value is set on SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO, it takes
+      # precedence over any SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION
+      SYSTEM_TEST_JS_STELLAR_SDK_GH_REPO:
```

Comment:
REPO needs to be populated, in this case it would be `https://github.com/stellar/js-stellar-sdk`, the conditional steps below are triggered by it's presence.

Reply:
oh okay got it, no defaults if tag is specified - fixed!

btw https://github.com/stellar/js-stellar-sdk/pull/877 will let us use an npm tag instead

Reply:
yes, if you want to wait for that to publish, then can just do `SYSTEM_TEST_JS_STELLAR_SDK_NPM_VERSION: 11.0.0-beta.6`, since sdk is changing often, will be changing this to a newer ref soon enough anyways, but there may be a little more value in verifying coverage against the npm repo. as it validates that published code is good.